anyway/parsers/waze/waze_db_functions.py

```diff
@@ -44,3 +54,15 @@ def _upsert_waze_objects_by_uuid(model, waze_objects):
 
         db.session.commit()
     return new_records
+
+
+def _enrich_ended_at_timestamp(model, timestamp, latest_waze_objects, back_filled):
```

Comment:
If this functions just updates the entries in the DB, then I assume it can be done directly in the DB, without reading the data to memory and then writing it. I do not know whether time matters here.